src/App.jsx

```diff
@@ -46,11 +47,30 @@ const App = () => {
 
       console.log("Airtable API Response Data", data);
 
-      const todos = data.records.map((record) => ({
+      //ascending order
+      const sortedData =data.records.sort((a,b) => {
+        const titleA = a.fields.title ? a.fields.title.toLowerCase() : '';
+        const titleB = b.fields.title ? b.fields.title.toLowerCase() : '';
+
+        if (isAscending){
+          return titleA < titleB ? -1 : titleA >titleB ? 1:0;
+        }else{
+          return titleA<titleB ?1: titleA > titleB ? -1 :0;
+        }
+
+      });
+
+
+      const todos = sortedData.map((record) => ({
         title: record.fields.title,
         id: record.id,
         task_time: record.fields.task_time || "",
       }));
+      // const todos = data.records.map((record) => ({
```

Comment:
Same as above - no need for a change in this review but going forward, make sure you're removing commented out code to keep your codebase clean.